editor.planx.uk/src/@planx/components/FileUploadAndLabel/schema.ts

```diff
@@ -31,6 +31,10 @@ const moreInformationSchema: SchemaOf<MoreInformation> = object({
   definitionImg: string(),
 });
 
+const baseNodeDataSchema: SchemaOf<BaseNodeData> = object({
+  tags: array(mixed().oneOf(["placeholder"])),
```

Comment:
nit: surprised to see `["placeholder"]` hardcoded specifically in this component file - can we define & export/import list of available tags someplace once?

Reply:
Good catch thanks - now added 👍